core/domain/feedback_jobs_one_off.py

```diff
@@ -100,3 +103,118 @@ def _cache_last_nonempty_message_author_id(
             thread_model.last_nonempty_message_author_id = message_author_id
             return True
         return False
+
+
+class FeedbackThreadIdRegenerateOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
I just found one more model GeneralFeedbackThreadUserModel which uses/stores thread_id. I'll have to add this to the job.

/cc @nithusha21